Client/mods/deathmatch/logic/lua/CLuaFunctionParseHelpers.cpp

```diff
@@ -653,21 +653,21 @@ ADD_ENUM(SURFACE_ADHESION_GROUP_SAND, "sand")
 ADD_ENUM(SURFACE_ADHESION_GROUP_WET, "wet")
 IMPLEMENT_ENUM_END("surface-adhesion-group")
 
-IMPLEMENT_ENUM_BEGIN(eCollisionShapes)
-ADD_ENUM(COLLISION_BOX, "box")
-ADD_ENUM(COLLISION_SPHERE, "sphere")
-ADD_ENUM(COLLISION_TRIANGLE, "triangle")
-ADD_ENUM(COLLISION_VERTEX, "vertex")
+IMPLEMENT_ENUM_CLASS_BEGIN(eCollisionShape)
+ADD_ENUM(eCollisionShape::BOX, "box")
+ADD_ENUM(eCollisionShape::SPHERE, "sphere")
+ADD_ENUM(eCollisionShape::TRIANGLE, "triangle")
+ADD_ENUM(eCollisionShape::VERTEX, "vertex")
 IMPLEMENT_ENUM_END("collision-shape")
```

Comment:
implement_enum_class_end

Reply:
still 
![image](https://user-images.githubusercontent.com/22455534/82762885-c971f280-9e03-11ea-95f1-1e79370a47fb.png)

Reply:
try including the header file, see if that helps
or try forward declaring(`enum class eCollisionKey;`)

Reply:
still same issue

Reply:
its some include order issue.